corehq/apps/users/views/__init__.py

```diff
@@ -1004,3 +1004,12 @@ def location_restriction_for_users(request, domain):
         project.location_restriction_for_users = json.loads(request.POST["restrict_users"])
     project.save()
     return HttpResponse()
+
+
+@require_POST
+@require_superuser
+def register_fcm_device_token(request, couch_user_id, device_token):
```

Comment:
It sounds fine to start this off as only available to superusers, but I imagine we'll eventually want any web user to be able to change their own FCM token.
@wspride @sjain28
Do you know of any analogous requests from mobile?  Something that doesn't yet have an application?  Trying to figure out how we should best handle auth.